src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -372,6 +440,7 @@ public void loadGestures(String directory) {
       }
     }
     info("%s Gestures loaded, %s Gestures with error", totalLoaded, totalError);
+    return ((totalError == 0) ? true : false);
```

Comment:
so. any errors means this function returns false...what reads the return value of this fuction?